pkg/wmiinstance/WmiClass.go

```diff
@@ -285,6 +285,16 @@ func (c *WmiClass) InvokeMethod(methodName string, methodParams []string, inputO
 	// both are SWbemObjects: https://docs.microsoft.com/en-us/windows/win32/wmisdk/swbemobject
 }
 
+func (c *WmiClass) InvokeMethodWithLiteralParams(methodName string, params ...interface{}) ([]interface{}, error) {
```

Comment:
Where are we planning to use this?

Reply:
We use it to create cluster group sets, before we didn't have a way to invoke static methods on wmi classes.

https://github.com/jocardeMSFT/moc-pkg/blob/13ffa1b226223a15f1eff1dfa64bb64695eff408/pkg/failovercluster/fc_avset_windows.go#L90C2-L90C183

Reply:
Also, how different is this from the unimplemented InvokeMethod call that is right above?

Reply:
In other words, let's just name this function "InvokeMethod"

Reply:
the method above takes a methodParams []string argument, this definition takes effectively an array of interfaces.

(one of the params on the call we invoke this is actually a string[] itself.

Reply:
replaced call, let me know if this was your intent @JocelynBerrendonner

Reply:
Thanks! Looks good.